test/actions/signMetadata.test.ts

```diff
@@ -0,0 +1,57 @@
+import { Keypair } from '@solana/web3.js';
+import axios from 'axios';
+import { Account, Connection, NodeWallet } from '../../src';
+import { mintNFT } from '../../src/actions';
+import { FEE_PAYER, pause } from '../utils';
+import { Metadata } from '../../src/programs/metadata';
+import { signMetadata } from '../../src/actions/signMetadata';
+import { mockAxios200, uri } from './shared';
+
+jest.mock('axios');
+jest.setTimeout(100000);
+
+describe('signing metadata on a master edition', () => {
```

Comment:
Does this test send requests to Solana's API?

Reply:
Hm not sure I understand the question.

If you're asking about RPC, then yes the test is running on devnet and so it's talking to Solana's RPC node. I modelled all the tests on the `mintNFT.test.ts`, which was doing the same.

If you're asking about axios mocking, then that's needed because inside of `mintNFT` we do an axios call to fill in the metadata, and we're mocking that call.